commit 26a49879f53b69a1aa91f5cd9678f7829f524845
Author: Aws Ismail <aws.ismail@windriver.com>
Date:   Thu Oct 25 13:43:17 2012 -0400

    Specify racoon2 submodules for configure
    
    Literally specify the submodules to configure
    instead of using the $SELECTED_MODULES variable
    lib, spmd, iked are always available. kinkd
    is selected if enabled and that depends
    on the availability of krb5 on the target.
    
    Signed-off-by: Aws Ismail <aws.ismail@windriver.com>

diff --git a/configure.in b/configure.in
index 3f83241..5eaa7c5 100644
--- a/configure.in
+++ b/configure.in
@@ -49,6 +49,9 @@ if test x$prefix = x"NONE" ; then
 	ac_configure_args="$ac_configure_args --prefix='$prefix'"
 fi
 
+AC_CONFIG_SUBDIRS(lib)
+AC_CONFIG_SUBDIRS(spmd)
+
 dnl Enable/disable each module
 AC_MSG_CHECKING(if --enable-iked option is specified)
 AC_ARG_ENABLE(iked, [  --enable-iked           build iked [yes]],
@@ -56,6 +59,7 @@ AC_ARG_ENABLE(iked, [  --enable-iked           build iked [yes]],
 AC_MSG_RESULT($enable_iked)
 if test x"$enable_iked" = xyes; then
 	SELECTED_MODULES="$SELECTED_MODULES iked"
+    AC_CONFIG_SUBDIRS(iked)
 fi
 AC_MSG_CHECKING(if --enable-kinkd option is specified)
 AC_ARG_ENABLE(kinkd, [  --enable-kinkd          build kinkd [yes]],
@@ -63,10 +67,8 @@ AC_ARG_ENABLE(kinkd, [  --enable-kinkd          build kinkd [yes]],
 AC_MSG_RESULT($enable_kinkd)
 if test x"$enable_kinkd" = xyes; then
 	SELECTED_MODULES="$SELECTED_MODULES kinkd"
+    AC_CONFIG_SUBDIRS(kinkd)
 fi
-dnl XXX Variables should not be used with AC_CONFIG_SUBDIRS on autoconf-2.5x,
-dnl but multiple/selective AC_CONFIG_SUBDIRSes don't work on autoconf-2.13.
-AC_CONFIG_SUBDIRS(lib $SELECTED_MODULES)
 
 dnl Switch for startup mechanism.
 AC_SUBST(startup_scripts)
